site/content/en/docs/Installation/_index.md

```diff
@@ -273,6 +338,28 @@ sudo az aks install-cli
 # Get credentials for your new AKS cluster
 az aks get-credentials --resource-group $AKS_RESOURCE_GROUP --name $AKS_NAME
 ```
+{{% /feature %}}
+{{% feature publishversion="1.2.0" %}}
+```bash
+# Declare necessary variables, modify them according to your needs
+AKS_RESOURCE_GROUP=akstestrg     # Name of the resource group your AKS cluster will be created in
+AKS_NAME=akstest     # Name of your AKS cluster
+AKS_LOCATION=westeurope     # Azure region in which you'll deploy your AKS cluster
+
+# Create the Resource Group where your AKS resource will be installed
+az group create --name $AKS_RESOURCE_GROUP --location $AKS_LOCATION
+
+# Create the AKS cluster - this might take some time. Type 'az aks create -h' to see all available options
+# The following command will create a single Node AKS cluster. Node size is Standard A1 v1 and Kubernetes version is 1.13.12. Plus, SSH keys will be generated for you, use --ssh-key-value to provide your values
+az aks create --resource-group $AKS_RESOURCE_GROUP --name $AKS_NAME --node-count 1 --generate-ssh-keys --node-vm-size Standard_A4_v2 --kubernetes-version 1.13.12
```

Comment:
This command has executed fine for me, I was able to install agones afterwards using install.yaml. The only issue I met is that I need to run `kubectl create namespace agones-system`.
At the end I have tested with creating simple-udp fleet.
Running E2E tests right now, I expect that we should increase node-count to 4 as in GKE cluster.

Reply:
E2E tests are failing for instance here:
```
FAIL: TestPingHTTP (30.11s)
    ping_test.go:50:
                Error Trace:    ping_test.go:50
                Error:          Expected nil, but got: &url.Error{Op:"Get", URL:"http://51.105.176.201:80", Err:(*net.OpError)(0xc00040a000)}
```

Reply:
I'm happy to increase the node count to 4. 

What is the IP address for that can't be fetched? Can you tell if it's because the cluster is too small and there are pending pods?

Reply:
I need to test one more time, could be connectivity issue between PC running E2E tests and the cluster.

Reply:
I followed all instructions for AKS cluster, issues does not related to this change.
Something is missing in description on how to create UDP firewall for port range. 
Therefore I think I would send a separate PR for the fix of docs to make E2E tests passing on both AKS and EKS.

Reply:
Thanks for testing this out @aLekSer!

Reply:
I've adjusted the AKS instructions to create a 4 node cluster instead of a 1 node cluster.